packages/styles/src/tokens/colors.ts

```diff
@@ -135,12 +135,10 @@ const categorical = {
 const common = {
   ...base,
   ...categorical,
+  // TODO: add "base palette" colors?
```

Comment:
If we spread the "base palette" colors here, they'll be injected as CSS variables and available in `theme.colors.*`, but these tokens do not vary depending on theme.
1. Should we inject the CSS vars?
2. Should we make them available only on the "runtime" `theme` object (from `useTheme`)?
3. Should we keep it as a separate static `import`?

`1.` seems wasteful, and if we don't want to promote the usage of the palette colors directly `3.` might make the most sense

Reply:
The "base palette" are the colors added to colosPalette.ts, right?

Doing only `2` means that users will only be able to use these colors inside a function component or hook (otherwise they'll need to "hack" their way into using them somewhere else). I'm not 100% comfortable with that.

We could do `3` but it seems a bit confusing to me (and to users) to import colors from two different places: `theme` and static imports. 

I agree that `1` seems wasteful, and we would be polluting the stylesheets with unnecessary CSS vars. However, it makes sense to have access to these colors when using `theme`. Can't we do a mix of both inside the `theme` object? Some colors are CSS vars and other ones are static values? For example we have utils like `spacing` or `alpha` that are static.

Reply:
Yeah, we could optimize which colors are injected as CSS vars 👍

Reply:
Tokens are available from `theme.palette` (alternative suggestions welcome)